src/views/v2/rsvp/details/availability.php

```diff
@@ -17,18 +17,23 @@
  * @version TBD
  */
 
-$is_unlimited = -1 === $rsvp->remaining();
+$is_unlimited     = - 1 === $rsvp->remaining();
+$is_in_stock      = $rsvp->is_in_stock();
+$has_days_to_rsvp = false;
 
+if ( $is_in_stock ) {
+	$days_to_rsvp     = $this->template( 'v2/rsvp/details/availability/days-to-rsvp', [ 'rsvp' => $rsvp ], false );
+	$has_days_to_rsvp = ! empty( $days_to_rsvp );
+}
 ?>
 <div class="tribe-tickets__rsvp-availability tribe-common-h6 tribe-common-h--alt tribe-common-b3--min-medium">
-	<?php if ( ! $rsvp->is_in_stock() ) : ?>
+	<?php if ( ! $is_in_stock ) : ?>
 		<?php $this->template( 'v2/rsvp/details/availability/full', [ 'rsvp' => $rsvp ] ); ?>
 	<?php elseif ( $is_unlimited ) : ?>
-		<?php $this->template( 'v2/rsvp/details/availability/unlimited', [ 'is_unlimited' => $is_unlimited ] ); ?>
+		<?php $this->template( 'v2/rsvp/details/availability/unlimited', [ 'is_unlimited' => $is_unlimited, 'has_days_to_rsvp' => $has_days_to_rsvp ] ); ?>
```

Comment:
:no_entry_sign: When a multi-item array uses associative keys, each value should start on a new line.

<sub>hash: 1b489f33698a7de22248069fb67cab59</sub>